docs/reference/query-rules/apis/put-query-ruleset.asciidoc

```diff
@@ -109,8 +109,8 @@ The following example creates a new query ruleset called `my-ruleset`.
 
 Two rules are associated with `my-ruleset`:
 
-- `my-rule1` will pin documents with IDs `id1` and `id2` when `user.query` exactly matches `marvel` _or_ `dc` **and** `user.country` exactly matches `us`.
-- `my-rule2` will pin documents from different, specified indices with IDs `id3` and `id4` when the `query_string` fuzzily matches `comic`.
+- `my-rule1` will pin documents with IDs `id1` and `id2` when `user_query` contains `pugs` _or_ `puggles` **and** `user_country` exactly matches `us`.
```

Comment:
Why the switch from dot notation here? (e.g. `user.query` -> `user_query`)

Reply:
The feedback was that all of our examples should be consistent. So since we were using `user_query` everywhere else, it was easier to update this one document over the others.